Makefile

```diff
@@ -3,10 +3,11 @@ all:
 	@echo "Use 'make install' to install whr930 on this system"
 
 install:
-	@cp src/whr930.py /usr/local/bin/whr930.py
+	@mkdir /opt/wtw
```

Comment:
Use `@mkdir -p /opt/wtw`

This overcomes 2 issues, if /opt does not exist, this will create it and if /opt/wtw already exists, it does not complain and continues